[2024-08-31T01:10:09.418+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-31T01:10:09.471+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: olist_dag.extract_data scheduled__2023-05-19T00:00:00+00:00 [queued]>
[2024-08-31T01:10:09.493+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: olist_dag.extract_data scheduled__2023-05-19T00:00:00+00:00 [queued]>
[2024-08-31T01:10:09.495+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-08-31T01:10:09.547+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_data> on 2023-05-19 00:00:00+00:00
[2024-08-31T01:10:09.591+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=1287) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-31T01:10:09.602+0000] {standard_task_runner.py:64} INFO - Started process 1309 to run task
[2024-08-31T01:10:09.603+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'olist_dag', 'extract_data', 'scheduled__2023-05-19T00:00:00+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/elt_dag.py', '--cfg-path', '/tmp/tmp6xgs72yz']
[2024-08-31T01:10:09.612+0000] {standard_task_runner.py:91} INFO - Job 126: Subtask extract_data
[2024-08-31T01:10:09.733+0000] {task_command.py:426} INFO - Running <TaskInstance: olist_dag.extract_data scheduled__2023-05-19T00:00:00+00:00 [running]> on host ad3e56ef573e
[2024-08-31T01:10:09.940+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='olist_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-19T00:00:00+00:00'
[2024-08-31T01:10:09.944+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-31T01:10:09.948+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-31T01:10:09.949+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/elt_dag.py", line 23, in extract_data
    from src.extract import extract
ModuleNotFoundError: No module named 'src'
[2024-08-31T01:10:09.976+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=olist_dag, task_id=extract_data, run_id=scheduled__2023-05-19T00:00:00+00:00, execution_date=20230519T000000, start_date=20240831T011009, end_date=20240831T011009
[2024-08-31T01:10:10.010+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 126 for task extract_data (No module named 'src'; 1309)
[2024-08-31T01:10:10.061+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-31T01:10:10.113+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-31T01:10:10.122+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
